internal/producer/src/accounts/accounts.go

```diff
@@ -454,6 +454,13 @@ func Equals(contract1 Contract, contract2 Contract) bool {
 	return true
 }
 
+// ContractToString takes in a Contract and return a string version
+func (c Contract) ToString() string {
+	return fmt.Sprintf("Version: %v\nSenderPubKey: %v\nSigLen: %v\nSignature: %v\nRecipPubKeyHash: %v\nValue: %v\nStateNonce: %v\n",
+						 c.Version, c.SenderPubKey, c.SigLen, hex.EncodeToString(c.Signature), hex.EncodeToString(c.RecipPubKeyHash), 
+						 c.Value, c.StateNonce)
```

Comment:
Can you utf-8 encode the `SenderPubKey` and then hex encode it?

Reply:
I read that utf-8 is default, so I tried to convert properly but wasn't sure - check it out